libseccomp/tests/tests.rs

```diff
@@ -287,6 +300,13 @@ fn test_builder_pattern() -> Result<(), Box<dyn std::error::Error>> {
         .set_api_sysrawrc(false)?
         .load()?;
 
+    #[cfg(github_actions)]
```

Comment:
This way they are never executed. Can you add the following to `build.rs` and change the `cfg`s  to `#[cfg(not(github_actions))]`.

```rust
const GITHUB_ACTIONS: &str =  "GITHUB_ACTIONS";

fn main() {
    println!("cargo:rerun-if-env-changed={}", GITHUB_ACTIONS);

    [...]    

    if env::var(GITHUB_ACTIONS).as_deref() == Ok("true") {
        println!("cargo:rustc-cfg=github_actions");
    }
}
```

Reply:
I have made the changes and pushed it. 
@rusty-snake please review it.